charts/velocity-operator/templates/manager-serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "chart.fullname" . }}-manager
+  labels:
+  {{- include "chart.labels" . | nindent 4 }}
+{{- if .Values.controllerManager.manager.saImagePullSecrets }}
+imagePullSecrets:
+{{- range .Values.controllerManager.manager.saImagePullSecrets }}
+  - name: {{ . }}
+{{- end}}
+{{- end }}
```

Comment:
you have great chance to provide service account from values as well:
```
// values.yaml
serviceAccount:
   existingName: "" # uses existing service account
   imagePullSecrets: [] # let users define whatever struct they want
```

in chart:
```
{{- if $.Values.controllerManager.manager.serviceAccount.existingName }}

{{- end }}
```